test/univariate/discrete/categorical.jl

```diff
@@ -124,4 +124,17 @@ end
     @test Categorical([0.5, 0.5]) ≈ Categorical([0.5f0, 0.5f0])
 end
 
+@testset "issue #832" begin
+    priorities = collect(1:1000) .* 1.
```

Comment:
```suggestion
    priorities = collect(Float64, 1:1000)
```

Reply:
I was copying directly from the issue for reproducibility, but I concur that the style you propose is better (and semantically interchangeable)